package.json

```diff
@@ -77,11 +77,11 @@
   },
   "devDependencies": {
     "@storybook/addon-actions": "^6.1.16",
-    "@storybook/addon-essentials": "^6.1.16",
+    "@storybook/addon-essentials": "^6.3.4",
```

Comment:
these changes should not be part of this PR

Reply:
I changed them back on the last commit